main.py

```diff
@@ -0,0 +1,16 @@
+from bot import Aria
+from os import environ
+from os.path import join, dirname
+from dotenv import load_dotenv
+
+# .envファイルからロード
+load_dotenv(verbose=True)
+
+dotenv_path = join(dirname(__file__), '.env')
+load_dotenv(dotenv_path)
```

Comment:
This is equivalent to:
```suggestion
load_dotenv()
```
when there is a `.env` file in the same directory with `__file__`.

Reply:
I see